### PR TITLE
Dockerfile with GPU support

### PR DESCRIPTION
**Pull request by RainierBarrett**
Addresses #28. Only one container b/c GPU-capable container should be still usable in CPU-only environments.

### CODE REVIEW

File: containers/dockerfile [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+RUN git clone https://github.com/cmelab/hoomd-organics && \
+    cd hoomd-organics && \
+    conda env update -n base -f environment-gpu.yml && \
+    conda clean --all --yes -f
+
+WORKDIR hoomd-organics
+
+RUN python -m pip install --user -e .
+
+USER icomse-software:icomse-software
```
[the remark this answers is not in the document]
Comment (RainierBarrett) [reply containers/dockerfile:19]: Yeah, it's mostly a security thing, just to set user to non-root when the container is opened. It's not strictly necessary though, you're right. Should we remove it?

**Comment by chrisjonesBSU** [comment]
I think this is ready to go, but we should figure out/merge https://github.com/cmelab/containers/pull/28 first.  Right now, this container will have a bunch of unnecessary stuff installed into it.

File: containers/dockerfile [hunk 2]
```diff
@@ -0,0 +1,17 @@
+FROM cmelab/gpuhoomd3conda:latest
```
Comment (chrisjonesBSU) [inline containers/dockerfile:1]: Do we have a gpuhoomd4conda docker file yet? We'll need one now that we updated things to use hoomd4

Comment (chrisjonesBSU) [reply containers/dockerfile:1]: We will have one once the PR in the containers repo is merged.

Comment (RainierBarrett) [reply containers/dockerfile:1]: Not yet. I'll throw one together though

**Pull request merged**